e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -73,33 +75,8 @@ public void When_ExtractRequestReceivedForPatientWithNoDocs_Expect_DatabaseToBeU
         var gpcAccessDocument = waitFor(() -> emptyDocumentTaskIsCreated(conversationId));
         assertThatNotDocumentsWereAdded(gpcAccessDocument);
 
-    }
-
-    private void assertThatExtractContinueMessageWasSent(Document ehrContinue) {
-        assertThat(ehrContinue).isNotEmpty();
-        assertThat(ehrContinue.get("received")).isNotNull();
-    }
-
-    private void assertThatInitialRecordWasCreated(String conversationId, Document ehrExtractStatus, String nhsNumber) {
-        assertThat(ehrExtractStatus).isNotNull();
-        assertThat(ehrExtractStatus.get("conversationId")).isEqualTo(conversationId);
-        assertThat(ehrExtractStatus.get("created")).isNotNull();
-        assertThat(ehrExtractStatus.get("updatedAt")).isNotNull();
-        var ehrRequest = (Document) ehrExtractStatus.get(EHR_REQUEST);
-        assertThat(ehrRequest.get("requestId")).isEqualTo(REQUEST_ID);
-        assertThat(ehrRequest.get("nhsNumber")).isEqualTo(nhsNumber);
-        assertThat(ehrRequest.get("fromPartyId")).isEqualTo(FROM_PARTY_ID);
-        assertThat(ehrRequest.get("toPartyId")).isEqualTo(TO_PARTY_ID);
-        assertThat(ehrRequest.get("fromAsid")).isEqualTo(FROM_ASID);
-        assertThat(ehrRequest.get("toAsid")).isEqualTo(TO_ASID);
-        assertThat(ehrRequest.get("fromOdsCode")).isEqualTo(FROM_ODS_CODE);
-        assertThat(ehrRequest.get("toOdsCode")).isEqualTo(TO_ODS_CODE);
-    }
-
-    private void assertThatAccessStructuredWasFetched(String conversationId, Document accessStructured) {
-        assertThat(accessStructured.get("objectName")).isEqualTo(conversationId + GPC_STRUCTURED_FILENAME_EXTENSION);
-        assertThat(accessStructured.get("accessedAt")).isNotNull();
-        assertThat(accessStructured.get("taskId")).isNotNull();
+        var ackToRequester = (Document) waitFor(() -> Mongo.findEhrExtractStatus(conversationId).get("ackToRequester"));
+        assertThatAcknowledgementToRequestWasSent(ackToRequester);
```

Comment:
Should this be `requester`?
```suggestion
        assertThatAcknowledgementToRequesterWasSent(ackToRequester);
```